target/linux/ipq40xx/image/Makefile

```diff
@@ -60,14 +60,19 @@ define Build/SenaoFW
 	@cp $@.new $@
 endef
 
-define Device/8dev_jalapeno
+define Device/8dev_jalapeno-common
 	$(call Device/FitImage)
 	$(call Device/UbiFit)
+	BLOCKSIZE := 128k
+	PAGESIZE := 2048
+endef
+TARGET_DEVICES += 8dev_jalapeno-common
```

Comment:
TARGET_DEVICES is only required for a real device. Drop this line for the common node here.

Reply:
Thanks, I completely missed this.
Pushed update to fix this.